### PR TITLE
Add tests.

### PR DESCRIPTION
**Pull request by Yeti-or**
@zxqfox

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    stream.on('data', function(file) {
+        expect(file.data).not.to.exist;
+    });
+    stream.on('end', done);
+
+    stream.write(new gulpUtil.File({
+        path: 'file.js',
+        contents: new Buffer('{}')
+    }));
```
Comment (zxqfox) [inline test.js]:
```js
stream
  .write(new File({
    // ...
  }))
  .end()
```
?

File: test.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+    stream.end();
+});
+
+it('should not eval simple object', function(done) {
+    var stream = gulpEval();
+
+    stream.on('data', function(file) {
+        expect(file.data).not.to.exist;
```
Comment (qfox) [inline test.js]: Can we make a helper for this? Like:

``` js
function passThru(content) {
  return new Promise(function(res, rej) {
    var stream = gulpEval();
    stream
      .once('data', function(file) {
        res(file.data);
      })
      .once('error', rej);

    (typeof content === 'string') && (content = {
      path: 'file.js',
      content: new Buffer(content)
    });

    stream.write(new gulpUtil.File(content)) || rej('Writing to stream failed');
    stream.end();
  }));
}
```

And then:

``` js
it('should parse object with iks sorok dva', () =>
  passThru('{x: 42}')
    .should.eventually.eql({x: 42}));
```

File: test.js (hunk 1 above)
Comment (Yeti-or) [inline test.js]: https://nodejs.org/api/stream.html#stream_writable_write_chunk_encoding_callback
Returns: <Boolean> true if the data was handled completely.

Comment (zxqfox) [inline test.js]: Sorree, u're right.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+sudo: false
+
+language: node_js
+
+matrix:
+  include:
+    - node_js: "4.4.0"
```
Comment (qfox) [inline .travis.yml]: LTS же 4.4.2 ? Почему просто 4 не поставить?

И давай 0.12 (если возможно) и 5 добавим?

File: package.json [hunk 4]
```diff
@@ -2,13 +2,30 @@
...
+  "author": "Vasiliy Loginevskiy <yeti.or@gmail.com>",
+  "license": "MIT",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "npm run lint && npm run specs",
+    "specs": "mocha test",
+    "lint": "jshint . && jscs .",
+    "cover": "istanbul cover _mocha -- test",
+    "coveralls": "npm i coveralls && npm run cover && cat ./coverage/lcov.info | coveralls"
```
Comment (qfox) [inline package.json:18]: do we really need this here?

Comment (Yeti-or) [inline package.json:18]: why not ? cool badges everybody loves them

Comment (qfox) [inline package.json:18]: I mean, why not to move it to .travis.yml? Do we need it in package.json?

**Comment by qfox** [comment]
@Yeti-or Жмет прям, я заберу?

**Comment by Yeti-or** [comment]
@zxqfox делаю прям

**Comment by Yeti-or** [comment]
closed in #5

**Pull request closed without merging**